### ISSUE TITLE
guests - some listed multiple times w different stubs

### ISSUE THREAD

**Issue by gerbrent** [JupiterBroadcasting/jupiterbroadcasting.com#158]
Some [Guests](https://jupiterbroadcasting.net/guests/) are listed multiple times, due perhaps to different stubs used in various Fireside listings.

Examples:

* Danielle Foré
   * https://jupiterbroadcasting.net/guests/dani
   * https://jupiterbroadcasting.net/guests/daniellefore

* Drew DeVore
   * https://jupiterbroadcasting.net/guests/drewdevore
   * https://jupiterbroadcasting.net/guests/drewofdoom

* Jim Salter
   * https://jupiterbroadcasting.net/guests/jim-salter
   * https://jupiterbroadcasting.net/guests/jimsalter

* Philip Müller
   * https://jupiterbroadcasting.net/guests/philip-muller
   * https://jupiterbroadcasting.net/guests/philipmuller
   
* Stuart Langridge
   * https://jupiterbroadcasting.net/guests/stuart-langridge
   * https://jupiterbroadcasting.net/guests/stuartlangridge
   
We should:
* [ ] consolidate profiles
* [ ] make sure the scraper wont simply recreate them
   * if so, perhaps a change in Fireside by JB is more appropriate?

**Comment by gerbrent** [comment]
Could relate to https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/issues/72#issuecomment-1179765579 and thus #75

**Comment by elreydetoda** [comment]
So, I haven't gotten to work on #110 yet, but when I do I'll try and also take a look at this. There is probably an easier solution in the code we could do, like have an `aliases` or `nicknames` field which accepts all those variations. Then we could always identify that person in the future as well in case their url name (slug) changed again.

That's just off the top of my head though, I've got to look at the code to find out if it's possible.

**Comment by elreydetoda** [comment]
It actually looks like @kbondarev did some map for host already (probably for something else, but that means there is mapping logic in there :upside_down_face: ), we'll probably be able to do something like I was mentioning above then :grin: 

https://github.com/JupiterBroadcasting/show-scraper/blob/9adc48d71962b5642f4d2c641d9409454e018383/config.yml#L39

**Comment by kbondarev** [comment]
> It actually looks like @kbondarev did some map for host already (probably for something else, but that means there is mapping logic in there upside_down_face ), we'll probably be able to do something like I was mentioning above then grin
> 
> https://github.com/JupiterBroadcasting/show-scraper/blob/9adc48d71962b5642f4d2c641d9409454e018383/config.yml#L39

Yes, that map is used for exactly that. Will need to identify all the guests with with multiple usernames/stubs and add them all to that map.

**Comment by kbondarev** [comment]
@gerbrent 
- #202 fixes this to an extent - It removed all the duplicates you've mentioned above (+  a few more that I've found).
- The actual content of the files should be consolidated. This should still be done here in #75 as described in [here]. (https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/issues/72#issuecomment-1179765579)

- In terms of the show-scraper:
  -  This was achieved using the `usernames_map` described above. I added more items to it, [see here](https://github.com/JupiterBroadcasting/show-scraper/blob/31dc20dc1936479f81053a85009c7bb5cecbfa01/config.yml#L38-L56).
  - In order to address this:
    > make sure the scraper wont simply recreate them
    
    I added a `data_dont_override` list in the [config file](https://github.com/JupiterBroadcasting/show-scraper/blob/31dc20dc1936479f81053a85009c7bb5cecbfa01/config.yml#L58-L64). Any data file (sponsor or people) that has been "consolidated" or modified manually should be added to that list. The scraper will not override these files!
